src/actions/pause/commands.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+
+import { getPause, getSelectedSource } from "../../selectors";
+import { getPausedPosition } from "../../utils/pause";
+import { getNextStep } from "../../workers/parser";
+import { addHiddenBreakpoint } from "../breakpoints";
+import { features } from "../../utils/prefs";
+
+import type { ThunkArgs } from "../types";
+type CommandType = "stepOver" | "stepIn" | "stepOut" | "resume";
+
+/**
+ * Debugger commands like stepOver, stepIn, stepUp
+ *
+ * @param string $0.type
+ * @memberof actions/pause
+ * @static
+ */
+export function command(type: CommandType) {
+  return async ({ dispatch, client }: ThunkArgs) => {
+    // execute debugger thread command e.g. stepIn, stepOver
+    dispatch({ type: "COMMAND", value: { type } });
```

Comment:
command data doesnt seem to be consumed anywhere? this might be replaced with an api call and then we can skip the redux cycle (since no front end data needs to be updated)